content/file/file.go

```diff
@@ -349,6 +349,29 @@ func (s *Store) Resolve(ctx context.Context, ref string) (ocispec.Descriptor, er
 	return s.resolver.Resolve(ctx, ref)
 }
 
+func (s *Store) Delete(ctx context.Context, target ocispec.Descriptor) error {
```

Comment:
Do we have use cases for deleting content in file store?

Reply:
https://github.com/oras-project/oras-go/pull/470#discussion_r1151976911